.buildkite/pipeline.yml

```diff
@@ -62,23 +62,47 @@ steps:
           packages.baseDAO-ligo-meta.exes.baseDAO-ligo-meta
         -c bats ./scripts/test.bats
 
-  - label: test
+  - label: test-local-chain-007
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8735"
     if: *not_scheduled
-    commands:
+    commands: &test
     - nix-build ci.nix -A packages.baseDAO.tests.baseDAO-test
-    # Emulator + local chain for Delphi protocol
+    - export TASTY_NETTEST_DATA_DIR="$(mktemp -d --tmpdir="$PWD")"
```

Comment:
`$PWD` :suspect:

Reply:
oh, right, here it needs to be updated as well